terraform/templates/ec2_user_data.sh

```diff
@@ -10,9 +10,24 @@ if [ -e /dev/nvme1n1 ]; then
 	cat >> /etc/fstab <<-EOF
 	/dev/nvme1n1  /data  ext4  defaults,noatime,nofail  0  2
 	EOF
-
 	mkdir /data
 	mount /data
+
+	# non-persistent storage is managed by Docker under data-root
+	if ! ${persistent} ; then
+		cat >> /etc/fstab <<-EOF
+		/dev/nvme1n1  /var/lib/docker/volumes  ext4  defaults,noatime,nofail  0  2
```

Comment:
How about `mkdir /data/docker` and `ln -sf /data/docker /var/lib/docker/volumes`?

Reply:
resolved off-chain :)